### PR TITLE
Improving tutorial documentation and code

### PR DESCRIPTION
**Pull request by KianYang-Lee**
1. Added instructions on adding postgres connection.
2. Modified proper SQL syntax.
3. Remove redundant lines when writing to CSV.
4. Added QUOTE argument for copy_expert

### CODE REVIEW

**Review by uranusjr — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

File: docs/apache-airflow/tutorial.rst [hunk 1]
```diff
@@ -377,73 +377,85 @@ We need to have docker and postgres installed.
...
+- Port: 5432
+
+After that, you can test your connection and if you followed all the steps correctly, it should show a success notification. Proceed with saving the connection and we are now ready write the DAG.
 
 Let's break this down into 2 steps: get data & merge data:
 
 .. code-block:: python
 
+  from airflow.decorators import dag, task
+  from airflow.hooks.postgres_hook import PostgresHook
```
Comment (kaxil) [inline docs/apache-airflow/tutorial.rst]:
```suggestion
  from airflow.hooks.postgres import PostgresHook
```

Comment (kaxil) [reply docs/apache-airflow/tutorial.rst]: Since the other import path is deprecated

**Review by kaxil — COMMENTED** [review]
Static checks are also failing - https://github.com/apache/airflow/runs/3993717209?check_suite_focus=true

File: docs/apache-airflow/tutorial.rst (hunk 1 above)
Comment (KianYang-Lee) [reply docs/apache-airflow/tutorial.rst]: Done

**Review by kaxil — APPROVED** [review]

**Pull request merged**